modules/apps/app-builder/app-builder-test/src/testFunctional/tests/AppBuilder.testcase

```diff
@@ -3036,4 +3036,47 @@ definition {
 		DetailsView.deleteEntry(itemName = "Changed");
 	}
 
+	@description = "This is a use case for LPS-95964"
+	@priority = "5"
+	test ViewFieldSetsInTableView {
+		property test.name.skip.portal.instance = "AppBuilder#ViewFieldSetsInTableView";
+		property portal.acceptance = "true";
+
+		var myFormViewName = "My Registration form";
+		var myTableViewName = "My display table";
+		var myAppName = "My Registration app";
+
+		AppBuilderAdmin.addCustomObject(
+			customObjectName = "my custom object",
+			isCreatingFormView = "true",
+			isCreatingViaPlusIcon = "true");
+
+		FormViewBuilder.addFieldByDoubleClick(fieldType = "Text");
+
+		FormViewBuilder.validateFieldLabel(fieldLabel = "Text");
+
+		Click(locator1 = "FormViewBuilder#RULE_BUILDER_GO_BACK");
+
+		FormViewBuilder.setUpTwoFieldSets(
+			fieldSetName = "Field Sets",
+			fieldType = "Text",
+			fieldType2 = "Numeric");
+
+		FormViewBuilder.goToFieldsetsTab();
+
+		FormViewBuilder.addFieldByDoubleClick(fieldType = "Field Sets");
+
+		FormViewBuilder.setName(formViewName = "Fields from Fieldsets");
+
+		FormViewBuilder.validateFieldLabel(fieldLabel = "Field Sets");
+
+		FormViewBuilder.save();
+
+		AppBuilderAdmin.openItem(itemName = "Fields from Fieldsets");
+
+		FormViewBuilder.goToFieldsetsTab();
+
+		TableViewBuilder.validateDisabledField(fieldType = "Field Sets");
```

Comment:
Hi @calvinctvo , after you save the Form View, you have to go the the "Table Views" tab and select to create a new Table. After that you will validate the following scenarios:
- All fields from the fieldset are displayed and available
- Add a field by drag-drop and also by double-clicking
- Remove a field from Table View

Reply:
Also, remember to run the format source command to fix the sf issue.